tests/tensorflow/nightly/classifier-resnetrs.libsonnet

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import 'common.libsonnet';
+local gpus = import 'templates/gpus.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local timeouts = import 'templates/timeouts.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+
+{
+  local resnetrs = common.ModelGardenTest {
+    modelName: 'classifier-resnetrs',
+    paramsOverride:: {
+      trainer: {
+        train_steps: error 'Must set `trainer.train_steps`',
+      },
+      task: {
+        train_data: {
+          input_path: '$(IMAGENET_DIR)/train*',
+        },
+        validation_data: {
+          input_path: '$(IMAGENET_DIR)/valid*',
+        },
+      },
+    },
+    command: [
+      'python3',
+      'official/vision/beta/train.py',
+      '--experiment=resnet_rs_imagenet',
+      '--mode=train_and_eval',
+      '--model_dir=$(MODEL_DIR)',
+      '--config_file=official/vision/beta/configs/experiments/image_classification/imagenet_resnetrs50_i160.yaml',
+      '--params_override=%s' % std.manifestYamlDoc(self.paramsOverride) + '\n',
+    ],
+  },
+  local functional = common.Functional {
+    paramsOverride+: {
+      trainer+: {
+        train_steps: 320,
+      },
+    },
+  },
+  local convergence = common.Convergence {
+    paramsOverride+: {
+      trainer+: {
+        train_steps: 109200,
+      },
+    },
+    regressionTestConfig+: {
+      metric_success_conditions+: {
+        'validation/accuracy_final': {
+          success_threshold: {
+            fixed_value: 0.79,
+          },
+          comparison: 'greater',
+        },
+      },
+    },
+  },
+
+  local gpu_common = {
+    local config = self,
+
+    paramsOverride+:: {
+      runtime+: {
+        distribution_strategy: 'mirrored',
+        loss_scale: 'dynamic',
+        num_gpus: config.accelerator.count,
+      },
+    },
+  },
+  local k80 = gpu_common {
+    local config = self,
+
+    paramsOverride+:: {
+      task+: {
+        train_data+: {
+          global_batch_size: 128,
+         },
```

Comment:
Could you remove one space here to fix the spacing? I believe this is breaking the presubmit checks

Reply:
Removed whitespace.